### ISSUE TITLE
Live USB with persistence is a much desired feature

### ISSUE THREAD

**Issue by intelliant01** [RechercheTech/mss#38]
Will be great to have it automated based on one of these methods -https://wiki.ubuntu.com/LiveUsbPendrivePersistent

Or let's have LiLi on MSS and employ it as such - https://www.howtogeek.com/howto/14912/create-a-persistent-bootable-ubuntu-usb-flash-drive/ .

**Comment by cyberorg** [comment]
See persistence option in README here: https://github.com/cyberorg/live-fat-stick

**Comment by cyberorg** [comment]
Creating persistent usb fails, run the following before creating the stick:
```
cp /home/mssadmin/Li-f-e.iso /home/mssadmin/Li-f-e-16.04.2-amd64.iso
live-fat-stick --ubuntu-persistent /home/mssadmin/Li-f-e-16.04.2-amd64.iso /dev/sdXY

By default 500M cow file is created, to create bigger size, before running live-fat-stick, run:

export cowsize=1000M



```

**Comment by intelliant01** [comment]
Will recommend a mv and ln -s. 

How can we ensure that the live-usb-gui uses a higher cowsize than the default 500m?

**Comment by intelliant01** [comment]
This shall work. Keeping it open until documented.

**Comment by cyberorg** [comment]
Fix for missing grub files
mssupdate https://recherchetech.github.io/mss/fixes/ 20170719-grubpc

**Comment by geets04** [comment]
Found solution for `live-grub-stick` , as cowsize was not set in root env.
But still found error in `live-fat-stick`. Pastebin link for [live-fat-stick](https://pastebin.com/VM3igAmH)
